var/spack/repos/builtin/packages/root/package.py

```diff
@@ -370,8 +370,8 @@ def cmake_args(self):
                 'ON' if '+postgres' in spec else 'OFF'),
             '-Dpythia6:BOOL=%s' % (
                 'ON' if '+pythia6' in spec else 'OFF'),
-            '-Dpythia8:BOOL=%s' % (
-                'ON' if '+pythia8' in spec else 'OFF'),
+            # '-Dpythia8:BOOL=%s' % ( - not suported by spack
+            #     'ON' if '+pythia8' in spec else 'OFF'),
```

Comment:
Please specify `-Dpythia8:BOOL=OFF` instead, otherwise the build will bomb if ROOT picks the wrong defaults (as seems to be the case on my machine).

Reply:
I'm seeing the same thing, so I concur with @HadrienG2 (Pythia8 is rather easy to build in my experience and it would be good to support this dependency when it comes).

Reply:
@HadrienG2 @graeme-a-stewart done